tensorflow_cloud/python/tests/integration/call_run_on_script_with_keras_ctl_test.py

```diff
@@ -11,14 +11,37 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Integration tests for calling tfc.run on script with keras."""
 
+import os
+import sys
+from unittest import mock
+import tensorflow as tf
 import tensorflow_cloud as tfc
 
-# MultiWorkerMirroredStrategy
-tfc.run(
-    entry_point="tensorflow_cloud/python/tests/testdata/mnist_example_using_ctl.py",
-    distribution_strategy=None,
-    worker_count=1,
-    requirements_txt="tensorflow_cloud/python/tests/testdata/requirements.txt",
-    stream_logs=True,
-)
+
+class TensorflowCloudOnScriptTest(tf.test.TestCase):
+    def setUp(self):
+        super(TensorflowCloudOnScriptTest, self).setUp()
+        self.test_data_path = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), "../testdata/"
+        )
+
+    @mock.patch.object(sys, "exit", autospec=True)
+    def test_MWMS_on_script(self, mock_exit):
+        mock_exit.side_effect = RuntimeError("exit called")
```

Comment:
This looks too hacky. 

asserting that sys.exit was called with 0 without raising any runtime error is a good enough start.

Reply:
Done.